File: PRAK205-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
import math
prot = input()
inprot = prot.split()
a, b = map(int, inprot)
c = math.sqrt(b**2-a**2)#rumus pythagoras untuk mencari alas
keliling = a + b + c
luas = (1/2) * c * a
print("Alas =", round(c), "cm")
print("Tinggi =", a, "cm")
print("Keliling =", round(keliling), "cm")
print("Luas =", round(luas), "cm^2")


a = int(input())
b = int(input())
c = math.sqrt(b**2-a**2)
keliling = a + b + c
luas = (1/2) * c * a
print("Alas =", round(c), "cm")
print("Tinggi =", a, "cm")
print("Keliling =", round(keliling), "cm")
print("Luas =", round(luas), "cm^2")

File: PRAK205-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>
#include <math.h>

int main(){
    int a, b;
    scanf("%d", &a);
    scanf("%d", &b);
    int c = sqrt(pow(b, 2) - pow(a, 2));
    int k = a + b + c; 
    int l = (c * a)/2; 
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    printf("Keliling = %d cm\n", k);
    printf("Luas = %d cm^2", l);

    return 0;
}